surveys/survey_template_data/survey_templates.json

```diff
@@ -1,23 +1,165 @@
 [
   {
-    "name": "Sample Survey Template Intercept",
+    "name": "Event Survey – Observational",
+    "type": "observational",
+    "questions": [
+      {
+        "plugin_data": "{\"label\": \"How many people do you see in each age range?\", \"detail_level\": \"basic\", \"help_text\": \"\", \"required\": false}",
+        "plugin_uid": "age_observational",
+        "position": 1
+      },
+      {
+        "plugin_data": "{\"label\": \"How many people do you see of each gender?\", \"help_text\": \"\", \"required\": false}",
+        "plugin_uid": "gender_observational",
+        "position": 2
+      },
+      {
+        "plugin_data": "{\"label\": \"How many people do you see engaged in each activity?\", \"detail_level\": \"basic\", \"help_text\": \"\", \"required\": false}",
+        "plugin_uid": "activity_observational",
+        "position": 3
+      },
+      {
+        "plugin_data": "{\"label\": \"How many people do you see in each size group?\", \"help_text\": \"\", \"required\": false}",
+        "plugin_uid": "groups_observational",
+        "position": 4
+      }
+    ]
+  },
+  {
+    "name": "Event Survey – Intercept",
     "type": "intercept",
     "questions": [
       {
-        "plugin_data": "{\"label\": \"What year did you move into your current address?\", \"age\": null, \"help_text\": \"\", \"required\": false}",
-        "plugin_uid": "household_tenure",
+        "plugin_data": "{\"label\": \"How did you hear about this event?\", \"choices\": \"Facebook\\r\\nTwitter\\r\\nInstagram\\r\\nEmail\\r\\nPosters/signs\\r\\nFriend/word of mouth\\r\\nOther\", \"help_text\": \"\", \"initial\": \"\", \"required\": false}",
+        "plugin_uid": "radio",
         "position": 1
+      },
+      {
+        "plugin_data": "{\"label\": \"On a scale from 1-5 (5 being the best), how would you rate this event?\", \"choices\": \"1\\r\\n2\\r\\n3\\r\\n4\\r\\n5\", \"help_text\": \"\", \"initial\": \"\", \"required\": false}",
+        "plugin_uid": "radio",
+        "position": 2
+      },
+      {
+        "plugin_data": "{\"label\": \"Based on your best guess, approximately how much money do you plan on spending at this event?\", \"help_text\": \"\", \"initial\": null, \"min_value\": null, \"max_value\": null, \"required\": false}",
+        "plugin_uid": "float",
+        "position": 3
+      },
+      {
+        "plugin_data": "{\"label\": \"What is the zip code of your home?\", \"help_text\": \"\", \"required\": false}",
+        "plugin_uid": "location_zip_home",
+        "position": 4
+      },
+      {
+        "plugin_data": "{\"label\": \"What is the zip code of your workplace?\", \"help_text\": \"\", \"required\": false}",
+        "plugin_uid": "location_zip_work",
+        "position": 5
+      },
+      {
+        "plugin_data": "{\"label\": \"What year did you move into your current address?\", \"help_text\": \"\", \"required\": false}",
+        "plugin_uid": "household_tenure",
```

Comment:
Tyler specified slightly different wording for these three location questions. Did you have a reason for altering them? I'm open to different wording, I just want to make sure I can communicate to him the reason we changed them.

Reply:
This is the wording in the built in questions, and he already said we should use that when his differ slightly!

Reply:
Oops, got it! Sorry for not reading closer 👍